src/table/virtualized-table/hooks/use-measure-text.ts

```diff
@@ -8,10 +8,14 @@ export interface MeasureTextHook {
 
 const MAGIC_DEFAULT_CHAR = 'M';
 
-export default function useMeasureText(fontSize: string | undefined, fontFamily: string | undefined): MeasureTextHook {
+export default function useMeasureText(
+  fontSize: string | undefined,
+  fontFamily: string | undefined,
+  boldText?: boolean
+): MeasureTextHook {
   const { estimateWidth, measureText } = useMemo((): MeasureTextHook => {
     const context = document.createElement('canvas').getContext('2d') as CanvasRenderingContext2D;
-    context.font = `${fontSize} ${fontFamily}`;
+    context.font = `${boldText ? '600' : ''} ${fontSize} ${fontFamily}`;
```

Comment:
Why `600`? We use `bold`, which corresponds to a value of 700, as a styling attribute in `HeaderCell.tsx`

Reply:
It actaully doesn't go through to the HeadCellContent. According to the designs it should be 600. I'll change the totals so that is 600 as well

Reply:
I noticed now that there is no difference on the default font family, which the designs are based on. The change is more consistent though